x-pack/plugins/security_solution/public/resolver/view/edge_line.tsx

```diff
@@ -45,7 +45,7 @@ const StyledElapsedTime = styled.div<StyledElapsedTime>`
   left: ${(props) => `${props.leftPct}%`};
   padding: 6px 8px;
   border-radius: 999px; // generate pill shape
-  transform: translate(-50%, -50%) rotateX(35deg);
```

Comment:
:information_source: idea to try later - maybe try putting a `perspective` rule on it.